modules/dkan/dkan_datastore/modules/dkan_datastore_api/README.md

```diff
@@ -229,3 +229,9 @@ http://example.com/api/dataset/search?resource_id[pop]=d3c099c6-1340-4ee5-b030-8
 
 Returns the country, population, squarekm and id for US and AR from datasets 11 and 13.
 
+### Caching
```

Comment:
Yes, didn't know this was live, I'll document there from now. Is it a dev or end user doc?